PurchasesCoreSwift/FoundationExtensions/DateFormatter+Extensions.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Created by Andrés Boedo on 7/29/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+
+    static let iso8601SecondsDateFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
+        return dateFormatter
+    }()
+
+    static func date(fromiso8601SecondsOrMillisecondsString maybeDateString: String?) -> Date? {
```

Comment:
just realized I messed up casing in the suggestion, it should have been `fromISO8601SecondsOrMillisecondsString`, sorry for the noise!!

Reply:
+1 for `fromISO8601SecondsOrMillisecondsString`